synapseclient/entity.py

```diff
@@ -640,6 +641,8 @@ def is_container(entity):
     """Test if an entity is a container (ie, a Project or a Folder)"""
     if 'concreteType' in entity:
         concreteType = entity['concreteType']
+    elif 'type' in entity:
+        concreteType = entity['type']
     elif isinstance(entity, collections.Mapping):
```

Comment:
The rest call for get children returns the concrete type in entity['type'] . This is why i added that line.